README.md

```diff
@@ -1,5 +1,9 @@
 # gRPC-Kotlin/JVM - An RPC library and framework
 
+[![Build Status](https://travis-ci.org/grpc/grpc-kotlin.svg?branch=master)](https://travis-ci.org/grpc/grpc-kotlin)
+[![grpc-kotlin-stub](https://img.shields.io/maven-central/v/io.grpc/grpc-kotlin-stub.svg?label=grpc-kotlin-stub)](https://search.maven.org/search?q=g:%22io.grpc%22%20AND%20a:%22grpc-kotlin-stub%22)
+[![protoc-gen-grpc-kotlin](https://img.shields.io/maven-central/v/io.grpc/protoc-gen-grpc-kotlin.svg?label=protoc-gen-kotlin)](https://search.maven.org/search?q=g:%22io.grpc%22%20AND%20a:%22protoc-gen-grpc-kotlin%22)
```

Comment:
I couldn't use the text `protoc-gen-grpc-kotlin` as the badge label icon, probably because it is too long. So I shortened it to `protoc-gen-kotlin`. Is that ok?

Maybe you'd prefer something with "..." in the label text to show that some text was elided?

Reply:
Oh, nm, it seems that the full text works. I'll update that now.